app/views/api/v1/match_conversations/close.json.jbuilder

```diff
@@ -0,0 +1,9 @@
+json.match_id       @match.id
+json.topic          @match.topic.id
+json.user           @match.other_party(current_user)
+if @match.user_a_id == current_user.id
+  json.last_logout  @match.last_logout_a 
```

Comment:
You can pass over the `last_logout_X` to this partial, so you don't have this conditional here and you keep it clean. Same goes for other jbuilders.